model/actors/miner/sector.go

```diff
@@ -80,26 +111,99 @@ func (msi *MinerSectorInfo) AsVersion(version model.Version) (interface{}, bool)
 	}
 }
 
-func (msi *MinerSectorInfo) Persist(ctx context.Context, s model.StorageBatch, version model.Version) error {
-	ctx, _ = tag.New(ctx, tag.Upsert(metrics.Table, "miner_sector_infos"))
+func (msi *MinerSectorInfoV7Plus) Persist(ctx context.Context, s model.StorageBatch, version model.Version) error {
+	ctx, _ = tag.New(ctx, tag.Upsert(metrics.Table, "miner_sector_infos_v7plus"))
 	stop := metrics.Timer(ctx, metrics.PersistDuration)
 	defer stop()
 
 	m, ok := msi.AsVersion(version)
 	if !ok {
-		return xerrors.Errorf("MinerSectorInfo not supported for schema version %s", version)
+		return xerrors.Errorf("MinerSectorInfoV7Plus not supported for schema version %s", version)
 	}
 
 	metrics.RecordCount(ctx, metrics.PersistModel, 1)
 	return s.PersistModel(ctx, m)
 }
 
 type (
-	MinerSectorInfoList []*MinerSectorInfo
+	MinerSectorInfoV7PlusList []*MinerSectorInfoV7Plus
 )
 
-func (ml MinerSectorInfoList) Persist(ctx context.Context, s model.StorageBatch, version model.Version) error {
-	ctx, span := otel.Tracer("").Start(ctx, "MinerSectorInfoList.Persist")
+func (ml MinerSectorInfoV7PlusList) Persist(ctx context.Context, s model.StorageBatch, version model.Version) error {
+	ctx, span := otel.Tracer("").Start(ctx, "MinerSectorInfoListV7Plus.Persist")
+	if span.IsRecording() {
+		span.SetAttributes(attribute.Int("count", len(ml)))
+	}
+	defer span.End()
+
+	ctx, _ = tag.New(ctx, tag.Upsert(metrics.Table, "miner_sector_infos_v7plus"))
+	stop := metrics.Timer(ctx, metrics.PersistDuration)
+	defer stop()
+
+	if len(ml) == 0 {
+		return nil
+	}
+
+	if version.Major == 1 {
```

Comment:
This doesn't look right. This uses the slow insert method if the major schema version is 1 (the current version).You only want to do this if the schema version is < 1 because it allows each individual model to be downgraded to the V0 version in `m.Persist`.

This is all irrelevant anyway since we deleted the V0 schema

Reply:
> This is all irrelevant anyway since we deleted the V0 schema

In that case could we just remove this all together?